Language/Variables/Data Types/String/Functions/length.adoc

```diff
@@ -34,7 +34,7 @@ Returns the length of the String, in characters. (Note that this doesn't include
 
 [float]
 === Returns
-The length of the String in characters.
+The length of the String in characters as an unsigned int.
```

Comment:
```suggestion
The length of the String in characters. Data type: `unsigned int`.
```

This is the established convention for documenting return types.

Example: https://github.com/arduino/reference-en/blob/master/Language/Functions/Advanced%20IO/pulseIn.adoc#returns
Which extrapolates on the official reference sample, which doesn't provide guidance re: return type (https://github.com/arduino/reference-en/issues/199), but does for parameter type:
https://github.com/arduino/reference-en/blob/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc#parameters